packages/helpermodules/utils/_thread_handler.py

```diff
@@ -23,7 +30,17 @@ def split_chunks(to_split, n):
 
         for thread in threads_chunk:
             if thread.is_alive():
-                log.error(f"{thread.name} konnte nicht innerhalb des Timeouts die Werte abfragen, die abgefragten "
-                          "Werte werden nicht in der Regelung verwendet.")
+                log.error(f"{thread.name} konnte nicht innerhalb des Timeouts abgearbeitet werden.")
                 not_finished_threads.append(thread.name)
     return not_finished_threads
+
+
+def thread_handler(thread: threading.Thread):
+    if is_thread_alive(thread.name):
+        log.error(f"Thread {thread.name} ist bereits aktiv und wird nicht erneut gestartet.")
+    else:
+        thread.start()
```

Comment:
In z.B. `phase_switch.py` wird nach dem Aufruf des `thread_handler()` immer eine Erfolgsmeldung in das Log geschrieben. Das ist unabhängig davon, ob der neue Thread wirklich gestartet wurde. Es wäre sinnvoll, wenn `thread_handler()` einen Status zurückgeben würde, um solche "Falschmeldungen" abzufangen.
```suggestion
def thread_handler(thread: threading.Thread) -> bool:
    if is_thread_alive(thread.name):
        log.error(f"Thread {thread.name} ist bereits aktiv und wird nicht erneut gestartet.")
        return False
    else:
        thread.start()
        return True
```